packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.test.js

```diff
@@ -126,9 +128,27 @@ describe('<KeyValuesTable>', () => {
     const el = wrapper.find('.ub-inline-block');
     expect(el.length).toBe(data.length);
     el.forEach((valueDiv, i) => {
-      if (data[i].key !== 'jsonkey') {
+      if (data[i].expected) {
         expect(valueDiv.html()).toMatch(data[i].expected);
       }
     });
   });
+
+  it('renders HTTP headers as multiple string spans', () => {
```

Comment:
This test feels very complicated & hard to maintain. Could we not use the `.expected` field above to define the exact output we want for the two new test cases?

Reply:
Sure. I was figuring out how the tests work and maybe got carried away. Will change it.